src/main/java/io/confluent/connect/jdbc/gp/gpload/GPLoadDataIngestionService.java

```diff
@@ -222,4 +223,27 @@ public static String checkGPloadOutputForErrors(ArrayList<String> output) {
         return errMsg;
     }
 
+    public static Boolean checkForGploadBinariesInPath(String greenplumHome) {
```

Comment:
This implementation seems incorrect

Reply:
When a utility isn't available in any path, we receive following message:

which gpload1
gpload1 not found

When found, we receive following message:

which gpload     
/usr/local/gpdb/bin/gpload

The path may vary. So I would recommend to use negative case, ie, if you found "not found" in the result, mark it not preset and raise proper exception.

Reply:
Feedback addressed. Please check